shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -23,6 +23,9 @@
 #import "flutter/shell/platform/darwin/ios/framework/Source/platform_message_response_darwin.h"
 #import "flutter/shell/platform/darwin/ios/platform_view_ios.h"
 
+constexpr int kMicrosecondsPerSecond = 1000 * 1000;
+constexpr CGFloat kScrollViewContentSize = 10.0;
```

Comment:
1x1 seems risky since technically a 1x1 shouldn't have an offset (unless fractional coordinates are allowed). I've moved it to 2x2, worse case scenario it represents 16bytes.